internal/handler/wallet/sign.go

```diff
@@ -53,6 +56,20 @@ func (s *walletSign) Handle(params operations.SignParams) middleware.Responder {
 		return resp
 	}
 
+	// Fetch the account information for the wallet using the massaClient
```

Comment:
you don't need massa client to get the address of the wallet.

Reply:
done